test/SlowTests/Issues/RavenDB-16934.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Operations;
+using Raven.Server.Config;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_16934 : RavenTestBase
+    {
+        public RavenDB_16934(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task CRUD_Operations()
+        {
+            using (var store = GetDocumentStore(new Options
+            {
+                ModifyDatabaseRecord = record =>
+                {
+                    record.Settings[RavenConfiguration.GetKey(x => x.Databases.QueryOperationTimeout)] = "5";
+                }
+            }))
+            {
+                var rnd = new Random();
```

Comment:
```suggestion
                var rnd = new Random(23123);
```

Let's make sure that this is consistent.

Reply:
If you want to have it randomized for some reason then better to use `[InlineDataWithRandomSeed]` arrtibute

Reply:
changed the test